.murdock

```diff
@@ -2,7 +2,6 @@
 
 : ${TEST_BOARDS_AVAILABLE:="esp32-wroom-32 samr21-xpro"}
 : ${TEST_BOARDS_LLVM_COMPILE:="iotlab-m3 native nrf52dk mulle nucleo-f401re samr21-xpro slstk3402a"}
-# we can't use '-' in the variable names, convert them to '_' to add new boards
```

Comment:
Why was this the case before?

Reply:
Since it is unrelated please remove in separete commit

Reply:
> Why was this the case before?

The first idea was to use the board name as key to access the array of applications, but it did not work in the CI. Now we just compare the name of the board to a string.


> Since it is unrelated please remove in separete commit

Will do

Reply:
Done